### PR TITLE
Price Toggle App

### PR DESCRIPTION
**Pull request by jason-ng-zq99**
Price Toggle App lets you see the price of three packages of a particular phone plan. 

Simple click the toggle button to toggle between viewing the monthly price or the annually price.

Desktop views:
image

image

Mobile view:
image

### CODE REVIEW

File: src/pages/FrontendMentor/PriceToggleJason/index.tsx
Comment (hui444) [inline src/pages/FrontendMentor/PriceToggleJason/index.tsx]: style with styled components instead

File: src/pages/FrontendMentor/PriceToggleJason/index.tsx
Comment (hui444) [inline src/pages/FrontendMentor/PriceToggleJason/index.tsx]: fyi; can also be written as `left` alone

**Review by hui444 — COMMENTED** [review]
\>>:( add description to pr!

some details are lacking
* hover state for toggle
* card looks too fat? idk cant rly put my finger on it.. may be bc of the button; too close to the line
* text overflow when viewport width is too small
Screenshot 2021-11-12 at 9 02 03 AM
* spacing between cards in mobile view is inconsistent
Screenshot 2021-11-12 at 9 02 58 AM

but code looks good!

File: src/pages/FrontendMentor/PriceToggleJason/index.tsx
Comment (jason-ng-zq99) [reply src/pages/FrontendMentor/PriceToggleJason/index.tsx]: Resolved in 5755d41

File: src/pages/FrontendMentor/PriceToggleJason/index.tsx
Comment (jason-ng-zq99) [reply src/pages/FrontendMentor/PriceToggleJason/index.tsx]: Resolved in 970d94d

File: src/components/PriceToggleJason/styles/HighlightPackageCard.styled.ts
Comment (hui444) [inline src/components/PriceToggleJason/styles/HighlightPackageCard.styled.ts:6]: is there a reason for 2 widths?

File: src/components/PriceToggleJason/styles/NormalPackageCard.styled.ts
Comment (hui444) [inline src/components/PriceToggleJason/styles/NormalPackageCard.styled.ts]: can take our border-radius since there will always be one; meaning `border-radius: ${.......`

File: src/components/PriceToggleJason/styles/NormalPackageCard.styled.ts
Comment (hui444) [inline src/components/PriceToggleJason/styles/NormalPackageCard.styled.ts]: hhahaa 1 or 2 would probably be sufficient

**Review by hui444 — APPROVED** [review]
not v sure about the duplicated widths but the website looks good! make sure u nvr delete any of the fonts in index.html!  

otherwise lgtm! deconflict if any before merging

File: src/components/PriceToggleJason/styles/HighlightPackageCard.styled.ts
Comment (jason-ng-zq99) [reply src/components/PriceToggleJason/styles/HighlightPackageCard.styled.ts:6]: Nope! Resolved in 7c17859

File: src/components/PriceToggleJason/styles/NormalPackageCard.styled.ts
Comment (jason-ng-zq99) [reply src/components/PriceToggleJason/styles/NormalPackageCard.styled.ts]: Resolved in c7d400b

File: src/components/PriceToggleJason/styles/NormalPackageCard.styled.ts
Comment (jason-ng-zq99) [reply src/components/PriceToggleJason/styles/NormalPackageCard.styled.ts]: Resolved in e8c8b98

**Pull request merged**
